### PR TITLE
{tools}[GCCcore/6.4.0,GCCcore/7.3.0,GCCcore/8.2.0] UnZip v6.0

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
nic406 - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/3d2b1363e94115921111c8f351dd0531 for a full test report.

**Comment by smoors** [comment]
that's a lot of patches!

**Comment by akesandgren** [comment]
Yeah, it's the patches from the Ubuntu packaging (well not all of them actually)

**Comment by smoors** [comment]
@boegel since your review is requested I'll wait for your approval before merging.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/35e1d9f7300073bd7b4fb3145d5fc25f for a full test report.

**Comment by boegel** [comment]
@akesandgren Can we stick to the usual naming scheme for patch files here, i.e. `UnZip-6.0_xxx.patch`, maybe combine all of these into a single patch file by `cat`'ing them into one?

**Comment by akesandgren** [comment]
Doh... forgot to fix that.

File: easybuild/easyconfigs/u/UnZip/UnZip-6.0-GCCcore-6.4.0.eb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    '08-allow-greater-hostver-values.patch',
+    '09-cve-2014-8139-crc-overflow.patch',
+    '10-cve-2014-8140-test-compr-eb.patch',
+    '11-cve-2014-8141-getzip64data.patch',
+    '12-cve-2014-9636-test-compr-eb.patch',
+    '13-remove-build-date.patch',
+    '14-cve-2015-7696.patch',
+    '15-cve-2015-7697.patch',
+    '16-fix-integer-underflow-csiz-decrypted.patch',
+    '20-unzip60-alt-iconv-utf8.patch',
```
Comment (boegel) [inline easybuild/easyconfigs/u/UnZip/UnZip-6.0-GCCcore-6.4.0.eb]: @akesandgren By means of reminder: I would like to see these combined into a single patch that follows the usual naming scheme for patches...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/894f7ce38df4fe42a1b1e368fdf40f95 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2625.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/af0ea1743122d29fa7a14672859365f5 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
node2102.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/78201db4bdb8c4d4805c4b490a73a5ed for a full test report.

**Pull request merged**